conans/client/graph/graph.py

```diff
@@ -5,6 +5,7 @@
 RECIPE_DOWNLOADED = "Downloaded"
 RECIPE_INCACHE = "Cache"  # The previously installed recipe in cache is being used
 RECIPE_UPDATED = "Updated"
+RECIPE_INCACHE_DATE_UPDATED = "Cache (Updated date)"
```

Comment:
This doesn't sound it should be a recipe state, but I still need to understand all the details.

Reply:
Ok, it is good atm, lets leave it.